src/lazy/expanded/compiler.rs

```diff
@@ -216,14 +216,14 @@ impl TemplateCompiler {
     /// to the template without interpretation. `(literal ...)` does not appear in the compiled
     /// template as there is nothing more for it to do at expansion time.
     pub fn compile_from_source(
-        context: EncodingContextRef<'_>,
+        active_macros: &MacroTable,
```

Comment:
🪧 All changes in this file are either:
1. Replacing the previously-passed-around `EncodingContextRef` with a `&MacroTable`.
  OR
2. Mechanical renaming of `Macro` (which is now a user-facing handle) to `MacroDef` (which continues to not be user-visible).